v1.1.0_release_notes.md

```diff
@@ -0,0 +1,59 @@
+## Breaking Changes.
+dcrpool v1.1.0 introduces breaking changes to the configuration, existing configurations will have to be updated accordingly. The breaking 
+changes made are:
+ 1. `--backuppass` has been renamed to `--adminpass`.
+
+ 2.  `--lastnperiod` and `--maxgentime` are now of type `time.Duration`.
```

Comment:
I usually recommend using `1.` on every item of a numbered list - markdown will increment the numbers as required. This reduces the chance of mistakes, and reduces manual labour and diffs if the list is changed, for example, adding/removing an item in the middle of the list.